frontend/src/Onboarding/OnboardingInfoScreen2.tsx

```diff
@@ -0,0 +1,545 @@
+import React from "react";
```

Comment:
why is there a onboardingInfoScreen and onboardingInfoScreen2? I think I understand for the SignUpScreen page, we made the old one SignUpScreenV2. Is this a similar situation or do we have two onboarding pages now?

Reply:
ah yes so basically onboardingInfoScreen2 is just the old version. Didn't wanna fuck shit up if we delete that right now

Reply:
ooo so there were no changes to it? It's just renamed right?

Reply:
yeee